src/Async.js

```diff
@@ -85,6 +86,9 @@ const Async = React.createClass({
 				cache: initCache(nextProps.cache),
 			});
 		}
+		if (nextProps.value !== this.props.value) {
+			this.loadOptions('');
```

Comment:
Yes, I see how that can work if you have access to your option in your load function. Really though, it shouldn't have to kick off a search at the beginning *at all* in order to work. You should be able to supply your initial value.

I've decided to work around this with a tighter UX. If my field is already valued, I put the value in a hidden field and I display the label text as static text. Then a toggle link allows me to clear my selection and search for a new one.